lib/auth.js

```diff
@@ -971,7 +971,7 @@ function gatherContext(req, res, next) {
 
     /*
      * We share these conditions with other systems as part of an auth token
-     * (e.g. marlin for jobs).
+     * (e.g. formerly marlin for jobs).
```

Comment:
Could just drop the ref to jobs. IIUC this auth "context" is used in libmanta calls to mahi.authorize().

Reply:
Sure.